src/CoWSwapEthFlow.sol

```diff
@@ -122,7 +122,19 @@ contract CoWSwapEthFlow is
     }
 
     /// @inheritdoc ICoWSwapEthFlow
-    function deleteOrder(EthFlowOrder.Data calldata order) external {
+    function deleteManyOrders(EthFlowOrder.Data[] calldata orderArray)
```

Comment:
Very small nit: my naming preference goes to `deleteOrders` to make the name shorter. However, feel free to keep the current one.